package.json

```diff
@@ -3,11 +3,13 @@
   "version": "2.3.1",
   "description": "Validates strings as commit messages",
   "main": "index.js",
-  "bin": {"commitplease": "./commit-msg-hook.js"},
+  "bin": {
+    "commitplease": "./commit-msg-hook.js"
+  },
   "scripts": {
     "install": "node install",
     "uninstall": "node uninstall",
-    "test": "standard && nodeunit test.js",
+    "test": "standard && babel test.js -o tmp.js && nodeunit tmp.js && rm tmp.js",
```

Comment:
Let's try to replace this with a direct call to node, while also loading the babel register module, something like this:
```
node -r 'babel-core/register' node_modules/nodeunit/bin/nodeunit test.js
```
Which doesn't work, failing with this:
```
/Users/jzaefferer/dev/jzaefferer/commitplease/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126
      if (!option) this.log.error("Unknown option: " + alias + "." + key, ReferenceError);
                           ^

TypeError: Cannot read property 'error' of undefined
```
Can you give that a try? Might be a simple fix.